listener/logs/logger.js

```diff
@@ -30,7 +33,7 @@ const WinstonLogger = createLogger({
     // Silent logger when running unit tests.
     silent: process.env.DISABLED_LOGGING === 'true',
     // Set level log level
-    level: LoggerLevel,
+    level: loggerLevel,
     // Set log format output in the log file
     format: format.combine(
         format.timestamp({format: 'YYYY-MM-DD HH:mm:ss'}),
```

Comment:
I tested with `LOG_LEVEL=error` (and it prints `INFO: Initialized Winston with level: error`), but I still see debug printouts in the console. Maybe because of `new transports.Console({level: 'debug'})`.

Reply:
Indeed. The log level on the transport takes precedence. We could remove those which then uses the global log level.

Or, for file logging we could do `warning` or `error`.

Reply:
Removed the separate log level and changed file logging to error.

In the deployment we should expose the logs to the host so that we do not lose them.